woocommerce/woocommerce-android
# [Customer Search] address design feedback

Closes: #9472


### Description

The PR addresses some of the feedbacks given by the design:
* Selection of the customer from the list makes it added to the order
* Placeholder values for email and name
* Added username to the list items
* Highlighting of the search matches, based on the search type (all/username/email/name)

w3JBReW1UJesCf02sj43kR-fi-168_17832

I'll update empty screen and search copy in the next PR

### Testing instructions

Run on both WC 8+ and WC 8- and notice the changes mentioned in the description of the PR

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/a3697b63-c8af-425c-a5bf-6641f49a95bc


https://github.com/woocommerce/woocommerce-android/assets/4923871/8edd63c8-ac21-4dd0-ae23-a73340ac16c9
